python/langsmith/utils.py

```diff
@@ -251,6 +251,25 @@ def is_base_message_like(obj: object) -> bool:
         ]
     )
 
+def get_tracer_project(return_default_value=True) -> Optional[str]:
```

Comment:
Do we want to return `None` if `tracing_is_enabled() is False`?

Reply:
I don't think so, reason being if we are planning to use this within LangChain and a user manually creates a tracer or uses the context manager to start tracing, that currently doesn't commit to any environment variables.